lib/ws-channel.js

```diff
@@ -309,8 +337,12 @@ WebsocketChannel.prototype.notify = function(message) {
 
 
 WebsocketChannel.prototype.request = function(message, callback) {
+  if (this._closed) {
+    setImmediate(callback.bind(null, new Error(this._closed)));
```

Comment:
You handle the lack of a callback a few lines down, shouldn't this have a similar guard? Alos, `setImmediate()` takes arguments as arguments.. so `setImmediate(callback, Error(this._closed));` might be clearer.